io.github.rinigus.OSMScoutServer.json

```diff
@@ -0,0 +1,255 @@
+{
+    "app-id": "io.github.rinigus.OSMScoutServer",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.9",
+    "sdk": "org.kde.Sdk",
+    "command": "osmscout-server",
+    "finish-args": [
+        "--filesystem=host",
```

Comment:
I know we already accepted a lot of apps doing it, but I wonder if we shouldn't take different approach of expecting users to use `flatpak override` instead. (Just a general thought, feel free to ignore it.)

Reply:
[Assuming that we talk about host permissions] Offline maps can be rather large and its of advantage to let users to choose storage path as they wish. Anything that you would put on the user wil end up on my issue tracker in most of the cases.

Reply:
If the application uses a normal Qt file chooser then users can import and export maps wherever they want without the permission. This ofc needs testing and *possibly* code changes depending on the UI workflow.

Reply:
I have done some testing before submitting. Without this permission I was stuck in the sandbox home. With it, I could put files anywhere. 

[hopefully I haven't missed any comment anymore and all is replied to].